bellows/zigbee/application.py

```diff
@@ -356,6 +356,8 @@ async def load_network_info(self, *, load_devices=False) -> None:
                     "flow_control": (
                         flow_control.name.lower() if flow_control is not None else None
                     ),
+                    # Z2M will not load EZSP backups without this internal key
+                    "ezspVersion": ezsp.ezsp_version,
```

Comment:
Is this in the correct place for Z2M now? Shouldn't this be located in `metadata.internal.ezspVersion` for Z2M, and not `metadata.ezsp.ezspVersion`? Reference: https://github.com/Koenkk/zigbee-herdsman/blob/47ae47dbb70dfae4fe38a2458fc0d0250165f0d5/src/utils/backup.ts#L95

Reply:
When exporting from the internal Zigpy `NetworkBackup` to the old JSON format, the `metadata` key contents are merged into `internal`: https://github.com/zigpy/zigpy/blob/1aba0f1c15af94120349c0ba18a58bd9ced8c484/zigpy/backups.py#L286-L320

We don't actually use `internal` in zigpy, it was renamed to `metadata`.